ignore warning/error when building with Intel C++ compiler by including -wd308 flag in $CXXFLAGS
author: Kenneth Hoste (HPC-UGent)
--- forecast/src/Makevars.orig	2015-05-12 11:56:10.000000000 +0200
+++ forecast/src/Makevars	2015-05-28 10:53:13.878100139 +0200
@@ -1,2 +1,4 @@
 ## Use the R_HOME indirection to support installations of multiple R version
 PKG_LIBS=$(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)
+# disable Intel C++ compiler (icpc) warning/error #308, to avoid 'member "std::complex<double>::_M_value" is inaccessible'
+PKG_CXXFLAGS = -wd308
